.github/workflows/security-scanning.yml

```
@@ -0,0 +1,401 @@
+
+      - name: Generate SBOM for ${{ matrix.service }}
+        run: |
+          syft docker:${{ env.ECR_REGISTRY }}/${{ env.REGISTRY_ALIAS }}/${{ matrix.service }}:${{ github.sha }} \
+            -o spdx-json \
+            > sbom-${{ matrix.service }}-spdx.json
+          syft docker:${{ env.ECR_REGISTRY }}/${{ env.REGISTRY_ALIAS }}/${{ matrix.service }}:${{ github.sha }} \
+            -o cyclonedx-json \
+            > sbom-${{ matrix.service }}-cyclonedx.json
+
+      - name: Upload SBOMs
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.8)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)